### PR TITLE
CBG-3324: Add startup cnf logging to DefaultDbConfig

### PR DESCRIPTION
**Pull request by mohammed-madi**
CBG-3324

Describe your PR here...
- Copy startup config logging to default db config, to be returned on GET /db/_config?include_runtime=true calls
- Add check on TestConfigsIncludeDefaults 

## Pre-review checklist
- [ ] Removed debug logging (`fmt.Print`, `log.Print`, ...)
- [ ] Logging sensitive data? Make sure it's tagged (e.g. `base.UD(docID)`, `base.MD(dbName)`)
- [ ] Updated relevant information in the API specifications (such as endpoint descriptions, schemas, ...) in `docs/api`

## Dependencies (if applicable)
- [ ] Link upstream PRs
- [ ] Update Go module dependencies when merged

## [Integration Tests](https://jenkins.sgwdev.com/job/SyncGateway-Integration/build?delay=0sec)
- [ ] `GSI=true,xattrs=true` https://jenkins.sgwdev.com/job/SyncGateway-Integration/1974/

### CODE REVIEW

File: rest/config_database.go [hunk 1]
```diff
@@ -83,6 +83,18 @@ func GenerateDatabaseConfigVersionID(previousRevID string, dbConfig *DbConfig) (
 	return hash, nil
 }
 
+func DefaultPerDBLogging(bootstrapLoggingCnf base.LoggingConfig) *DbLoggingConfig {
+	if bootstrapLoggingCnf.Console != nil {
```
Comment (bbrks) [inline rest/config_database.go:87]: Do we want an `Enabled==false` check too?

If you have `LogLevel=Debug`/`LogKeys=["*"]` but `Console.Enabled=false` nothing will be logged.

**Review by bbrks — CHANGES_REQUESTED** [review]

File: rest/config_database.go (hunk 1 above)
Comment (mohammed-madi) [reply rest/config_database.go:87]: https://github.com/couchbase/sync_gateway/pull/6381/commits/99b323505babb6b6a70a6b59ebb11041763856b8 this good?

File: rest/adminapitest/admin_api_test.go [hunk 2]
```diff
@@ -563,11 +563,40 @@ func TestDBGetConfigNames(t *testing.T) {
...
+	emptyCnf := &rest.DbLoggingConfig{}
+	assert.Equal(t, body.Logging, emptyCnf)
 
 	for k, v := range body.Users {
 		assert.Equal(t, k, *v.Name)
 	}
+}
+
+// Tests that the users returned in the config endpoint have the correct names
+// Reproduces #2223
```
Comment (bbrks) [inline rest/adminapitest/admin_api_test.go]: You may want to update or just remove this ^

**Review by bbrks — CHANGES_REQUESTED** [review]
Looks good. Nitpick on copy/pasted test comment :)

**Review by bbrks — APPROVED** [review]

**Pull request merged**